openapi/components/schemas/PaymentInstruments/AlternativePaymentInstrument.yaml

```
@@ -0,0 +1,55 @@
+type: object
+title: 'Alternative instrument'
+properties:
+  id:
+    description: The payment instrument ID.
+    allOf:
+      - $ref: ../ResourceId.yaml
+  method:
+    description: The method of payment instrument.
+    allOf:
+      - $ref: ../PaymentMethods/AlternativePaymentMethods.yaml
+  customerId:
+    description: Customer's ID.
+    allOf:
+      - $ref: ../ResourceId.yaml
+  billingAddress:
+    description: The billing address.
+    allOf:
+      - $ref: ../Contact/ContactObject.yaml
+  status:
+    description: The payment instrument status.
+    type: string
+    enum:
+      - active
```

Can't it be deactivated?

---

I was not sure what's the point in deactivation of this almost dummy instrument, but maybe yes, I see that one may want to deactivate it to avoid using it as some default.
```suggestion
      - active
      - deactivated
```